labellines/core.py

```diff
@@ -162,7 +166,9 @@ def labelLines(lines, align=True, xvals=None, drop_label=False, shrink_factor=0.
                      for x in xvals]
     
     txts = []
-    for line, x, label in zip(labLines, xvals, labels):
-        txts.append(labelLine(line, x, label, align, drop_label, **kwargs))
+    if type(yoffsets) == float:
+        yoffsets = [yoffsets]*len(labLines)
```

Comment:
I'd suggest that instead, otherwise your test will fail with `yoffsets = 1` (which is an int).
```suggestion
    try:
        yoffsets = [float(yoffsets)]*len(labLines)
    except TypeError:
        pass
```